posts/templates/posts/_replied_posts.html

```diff
@@ -0,0 +1,40 @@
+{% load parser %}
```

Comment:
Was trying to explain with that the reason why I didn't use the same _post interface